Makefile

```diff
@@ -14,4 +14,7 @@ website:
 		npm start
 
 .DEFAULT_GOAL := website
-.PHONY: website
\ No newline at end of file
+.PHONY: website
+
+website-preview:
+	scripts/website-preview.sh
```

Comment:
I kinda like the script convention here, as opposed to inlining everything in the makefile. Goal: can we support both `make website` (published image, no docker build) and `make website/local` (builds local image from terraform-website repo) with one script and adjusting inputs with env vars?

Reply:
An alternative is trying to leverage env vars further in the makefile so we end up with something like this:

https://github.com/hashicorp/waypoint/blob/main/website/Makefile

Reply:
Something like the above makes it very easy roll it out to other projects as well, as all we need to do is change one env var

Reply:
> Goal: can we support both make website (published image, no docker build) and make website/local

Yea I can add support for both!

💭 To make the script _portable_, I could expose the various hardcoded variables as flags.
- The script could be generic, and copy-pasted around.
- The `Makefile`s would pass in repo-specific env vars as flags
  - This could help keep the `Makefile`s a bit leaner

Reply:
🤔 Not yet sure if that'll get unwieldy or not

Reply:
Ok, so going to go for a 100% Makefile alternative